package.json

```diff
@@ -6,6 +6,10 @@
   "scripts": {
     "build:dev": "webpack --node-env development",
     "build:prod": "webpack --config webpack.prod.js",
+    "cypress:run": "cypress run",
+    "cypress:headed": "cypress run --headed",
```

Comment:
For CI/CD purposes we would normally run these tests headlessly. We should leave the headed target here but for github actions, we should just run cypress:run

Reply:
I had that originally. This was a proposed work-around that I had tried. Apparently it had worked for some people. I'll put it back